django/contrib/contenttypes/fields.py

```diff
@@ -21,19 +21,33 @@ class GenericForeignKey(object):
     Provides a generic relation to any object through content-type/object-id
     fields.
     """
+    concrete = False
+    editable = False
+    hidden = False
+    auto_created = False
+
+    is_relation = True
+    one_to_many = True
+    many_to_many = False
+    many_to_one = False
+    one_to_one = False
+    related_model = None
 
     def __init__(self, ct_field="content_type", fk_field="object_id", for_concrete_model=True):
         self.ct_field = ct_field
         self.fk_field = fk_field
         self.for_concrete_model = for_concrete_model
+        self.generate_reverse_relation = False
```

Comment:
`generate_reverse_relation` seems unused now? (also defined in `db/models/fields/related.py`)